### DISCUSSION TITLE
Inquiry About Image Generation For Two Grayscale Images

### DISCUSSION THREAD

**Discussion by ffbh123456** [SixLabors/ImageSharp#2833] [Q&A] [answered]
I have two grayscale images, and I would like to know if the code in this repository could generate an image similar to the one I provided.
Could you please let me know if this is feasible with the current code? If so, any guidance on how to achieve this would be greatly appreciated.

<img width="634" alt="Screenshot 2024-10-29 at 7 42 07 PM" src="https://github.com/user-attachments/assets/02f4a317-6631-4ef1-9337-b3487e9d62f0">
<img width="768" alt="Screenshot 2024-10-29 at 7 42 24 PM" src="https://github.com/user-attachments/assets/8152e3a5-88c8-4fc3-a140-d2d5bad08283">

**Comment by JimBobSquarePants** [comment]
>I have two grayscale images, and I would like to know if the code in this repository could generate an image similar to the one I provided.

What do you actually mean by this? Do you want grayscale images or do you want algorithms to render a specific pattern or what?

**Reply by ffbh123456** [reply]
Hello, thank you for your reply. I would like to generate a grayscale image at runtime. For example, given the texture width, texture height, and circle radius as inputs, the output should be an image featuring a white circle, similar to the one I provided.

**Reply by JimBobSquarePants** [reply, marked as answer]
ImageSharp provides you with the means to create and manipulate image buffers however you would have to implement whatever algorithm you require to create the effect yourself.

**Reply by ffbh123456** [reply]
Understood. I'm not sure which algorithm to use for this effect at the moment. If you have any suggestions, I'd appreciate your input
